frontend/packages/topology/src/__tests__/TopologyShortcuts.spec.tsx

```diff
@@ -0,0 +1,136 @@
+import { shallow } from 'enzyme';
+import { Shortcut } from '@console/shared';
+import { TopologyViewType } from '../topology-types';
+import { getTopologyShortcuts } from '../components/graph-view/TopologyShortcuts';
+
+describe('TopologyShortcuts tests', () => {
+  it('should show reduced list in view shortcuts popover on topology toolbar when there are no workloads', () => {
+    const wrapper = shallow(
+      getTopologyShortcuts(jest.fn(), {
+        supportedFileTypes: undefined,
+        isEmptyModel: true,
+        viewType: TopologyViewType.graph,
+      }),
+    );
+    const secondWrapper = shallow(wrapper.getElement());
```

Comment:
why do we need a second shallow rendering here?

Reply:
I think because `shallow` rendering prevented her from reaching the details she wanted. `mount` would have been needed.